docs/dev/building-on-zksync/contracts/differences-with-ethereum.md

```diff
@@ -4,14 +4,13 @@ zkSync Era can handle almost all smart contracts based on the Ethereum Virtual M
 
 ## Differences
 
-### Using `call` over `.send` or `.transfer`
+### Use `call` over `.send`/`.transfer`
 
-The utilization of `payable(X).send` or `payable(X).transfer` is not feasible, as the 2300 gas stipend is generally insufficient for conducting a transfer. Furthermore, owing to diverse opcode pricing, the transfer may necessitate state changes, resulting in a large number of L2 gas being expended on data availability.
-
-To ensure adherence to security best practices, it is recommended to employ `call` instead. For instance:
+Avoid using `payable(X).send`/`payable(X).transfer` because the 2300 gas stipend may not be enough to conduct a transfer, especially if it involves state changes requiring a large amount of L2 gas spent on data availability. Instead, it is recommended to use `call`, which follows security best practices.
```

Comment:
```suggestion
Avoid using `payable(X).send`/`payable(X).transfer` because the 2300 gas stipend may not be enough to send a transfer, especially if it involves state changes requiring a large amount of L2 gas spent on data availability. Instead, it is recommended to use `call`, which follows security best practices.
```

Reply:
If you're going to mention security risks as important, say what/why/how.